Add notes on the PKI cert generation forwarding regression (#13815)
hashicorp/vault: website/content/partials/pki-forwarding-bug.mdx

## PKI Certificate Generation Forwarding Regression

> A bug introduced in Vault 1.8 causes certificates issue by the PKI secrets engine made on a performance

Reviewer: ```suggestion
A bug introduced in Vault 1.8 causes certificates generation requests via the PKI secrets engine made on a performance
```

Reviewer 2: I think "causes certificate generation requests via" (get rid of the pluralization of "certificates")?

> secondary node to be forwarded to a Vault cluster's primary node.  This affects horizontal scalability, but

Reviewer: I don't expect the horizontal scalability is the real issue most users, it's more about the certs/CRLs being in the wrong place.  If we do want to retain this bit, I might expand it a little - not everyone's going to follow what we mean by horizontal scalability.

Author: Okay, I'll just drop that part.
